docs/en/releases/main.md

```diff
@@ -2,6 +2,17 @@
 
 <Badge type="danger" text="Alpha" />
 
+<script setup>
```

Comment:
FYI, so I added this so that on this branch we should get a link to the actual "main" release notes. But we probably still should delete the rest of the content when we're done from here.